dev/index.less

```diff
@@ -181,7 +181,7 @@ a {
 
       #current-clue-label {
         display: inline-block;
-        width: @clue-label-width;
```

Comment:
Please preserve the less variable reference.

Reply:
Done - as you mentioned in another comment, I accidentally deleted the other usage of this so thought it was only used in one place.